### DISCUSSION TITLE
Distro poll

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#2174] [Show and tell]
![schermata-01-07-18-16](https://user-images.githubusercontent.com/10681413/148581929-64e85493-7b6a-4e75-b9ee-f49b4a0d453a.png)

Twitter doesn't allow more than 4 answers - maybe Suse instead of Fedora would have been better. 
https://twitter.com/lxqt_project/status/1478640826909663232

**Comment by tsujan** [comment]
> maybe Suse instead of Fedora would have been better.

Yes. Tumbleweed's LXQt is always up-to-date.

**Comment by dglent** [comment]
I use Mageia (i don't have twitter account)

**Reply by subhaniminhas** [reply]
I have lxqt as the only flavor in PakOS , based on Debian (https://sourceforge.net/projects/pakos/)
Unfortunately, since there is no advanced menu available in lxqt, so by default I have disabled lxqt menu, and enabled xfce menu. If there was a default advance menu in lxqt, it would have been my choice end to end. Other than menu, all underlying components are lxqt.

**Reply by stefonarch** [reply]
You mean the "whisker menu" I suppose. 
Atm the equivalent of "favorites" can only  be done configuring a quicklaunch plugin in an auto-hiding panel or similar, the other features are more or less the same IMO. Sure to have an "swiss knife menu" as a choice would be a nice thing - waiting for coders...

**Reply by edwardp3** [reply]
Mageia

**Comment by stefonarch** [comment]
Final results:

![schermata-01-12-11-47](https://user-images.githubusercontent.com/10681413/149126500-a0ca0bd9-b1fe-47e6-ab40-949a9211dd29.png)
